arangod/Agency/Agent.cpp

```diff
@@ -50,6 +50,24 @@ using namespace arangodb::application_features;
 using namespace arangodb::velocypack;
 using namespace std::chrono;
 
+
+struct AppendScale {
+  static log_scale_t<float> scale() { return {2.f, 0.f, 1000.f, 10}; }
+};
+struct AgentScale {
+  static log_scale_t<float> scale() { return {1.f, 0.f, 200.f, 10}; }
```

Comment:
The old scale definition was different:
```suggestion
  static log_scale_t<float> scale() { return {std::exp(1.f), 0.f, 200.f, 10}; }
```